### PR TITLE
docs: Llama Index OpenAI Agent tutorial edits

### PR DESCRIPTION
**Pull request by michaelschiff**

### CODE REVIEW

**Comment by michaelschiff** [comment]
I have read the CLA Document and I hereby sign the CLA

File: tutorials/tracing/llama_index_openai_agent_tracing_tutorial.ipynb [hunk 1]
```diff
@@ -2,7 +2,11 @@
  "cells": [
```
Comment (fjcasti1) [inline tutorials/tracing/llama_index_openai_agent_tracing_tutorial.ipynb:2]: <p>Is it worth requiring the pandas import just for that column width? I have run the colab with and without it and there is barely a difference</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/1466/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (fjcasti1) [inline tutorials/tracing/llama_index_openai_agent_tracing_tutorial.ipynb:2]: <p>Is it worth requiring the pandas import just for that column width? I have run the colab with and without it and there is barely a difference</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/1466/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (mikeldking) [reply tutorials/tracing/llama_index_openai_agent_tracing_tutorial.ipynb:2]: Feel free to nix

**Review by mikeldking — APPROVED** [review]

**Review by axiomofjoy — APPROVED** [review]

**Comment by mikeldking** [comment]
Thanks for the fixes!

**Pull request merged**